MapleServer2/Commands/Game/EventCommands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using MapleServer2.Commands.Core;
+using MapleServer2.Servers.Game;
+using MapleServer2.Types;
+
+namespace MapleServer2.Commands.Game
+{
+    public class EventCommands : InGameCommand
+    {
+        public EventCommands()
+        {
+            Aliases = new[]
+            {
+                "event"
+            };
+            Description = "Create a global event";
+            AddParameter<string[]>("id", "Event Id");
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+            string[] args = trigger.Get<string[]>("id");
+
+            if (args == null || args.Length <= 1)
+            {
+                trigger.Session.SendNotice("No events provided. Choose 3 events. 1 = OX Quiz \n" +
```

Comment:
Can we just have a 13 element string array to nicely display this? Could include just the text, like `Trap Master`, then use the index to count it, and then join the array using `\n`